[2024-05-04T11:40:58.697+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:40:58.759+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:40:58.815+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:40:58.815+0000] {taskinstance.py:2303} INFO - Starting attempt 4 of 4
[2024-05-04T11:40:58.889+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToGCSOperator): upload_file_sellers_to_capstone> on 2024-05-03 00:00:00+00:00
[2024-05-04T11:40:58.919+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12413) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T11:40:58.920+0000] {standard_task_runner.py:63} INFO - Started process 12473 to run task
[2024-05-04T11:40:58.926+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_sellers_to_capstone', 'scheduled__2024-05-03T00:00:00+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpwyeztxfp']
[2024-05-04T11:40:58.929+0000] {standard_task_runner.py:91} INFO - Job 200: Subtask upload_file_sellers_to_capstone
[2024-05-04T11:40:59.120+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [running]> on host d4c1d9b975b8
[2024-05-04T11:41:00.759+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_sellers_to_capstone' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T00:00:00+00:00'
[2024-05-04T11:41:00.760+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T11:41:00.843+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-04T11:41:00.877+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-04T11:41:02.151+0000] {gcs_to_gcs.py:535} INFO - Executing copy of gs://raw_data_projectcapstone/olist_sellers_dataset.csv to gs://storage-capstone/olist_sellers_dataset.csv
[2024-05-04T11:41:02.231+0000] {gcs.py:256} INFO - Total Bytes: 174703 | Bytes Written: 174703
[2024-05-04T11:41:02.231+0000] {gcs.py:264} INFO - Object olist_sellers_dataset.csv in bucket raw_data_projectcapstone rewritten to object olist_sellers_dataset.csv in bucket storage-capstone
[2024-05-04T11:41:02.231+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T11:41:02.241+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_sellers_to_capstone, execution_date=20240503T000000, start_date=20240504T114058, end_date=20240504T114102
[2024-05-04T11:41:02.391+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T11:41:03.305+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:41:03.350+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
